toss/udp_initial_condition.py

```diff
@@ -40,12 +37,13 @@ def __init__(self, args, lower_bounds, upper_bounds):
                     rtol (float): Relative error tolerance for integration.
                     atol (float): Absolute error tolerance for integration.
                 problem: Parameters related to the problem:
-                    start_time (float): Start time (in seconds) for the integration of trajectory.
-                    final_time (float): Final time (in seconds) for the integration of trajectory.
+                    start_time (int): Start time (in seconds) for the integration of trajectory.
```

Comment:
DEsolver DP8713M did not seem to manage decimal values for the boundaries on the integrated time-interval, in fact it resulted in the previously mentioned "infinity loop" of integration without indicating any errors. I have not evaluated the number of decimals it can manage, but integer values works fine. As our time axis is defined in seconds, maybe the effect of a shift in time to nearest integer would not be too great on the optimization with respect to the dynamics of the problem. In fact, if it is not, then maybe we could even let the search for the time of maneuvers to be an integer optimization process? In that case, it would at least significantly minimize the search space for these values.

Reply:
Well, you can discretize anything ofc but it is always a trade-off. Doesn't necessarily make it easier because smoothness etc can suffer and discretization of different variables has varying degrees of effect on the granularity you are searching (e.g. 1m in x is negligible. 1m/s in v is huge.)

But yea if it has to be like this ok, but maybe add a comment in the code explaining this so we don't wonder again in the future ;)

Reply:
Still open, I think?

Reply:
I added the comment explaining this in trajectory_tools.py (where all of the integration times are change to integer type). 

Maybe it would be a good idea to create another issue regarding integer optimization where we can also verify  whether desolver can manage floating points or not? Then, if studying desolver shows that we must use integers for some reason, it would be nice to maybe also use an optimizer with support for integer optimization, but that choice should reasonably be done at the same time as choosing one with capabilities for constrained optimization in issue #37